pandapipes/io/io_utils.py

```diff
@@ -62,8 +51,9 @@ def pandapipesNet(self):
             from pandapipes import from_json_string
             return from_json_string(self.obj)
         else:
-            self.net.update(self.obj)
-            return self.net
+            net = create_empty_network()
```

Comment:
I think this part is still somewhat unnecessary, but we could leave it inside for now. We should check, if pandapipesNet.__new__() can be called and the net.update(self.obj) leads to the same results. Missing tables and columns from earlier versions should be added in the convert_format script.

Reply:
Cool, never thought about that. We definitely should check that.

What do you mean by convert_format comment. What tables and columns do you have in mind, can't think of any right now.

Reply:
If we add a new table in future (e.g. "compressor"), it will not appear in old versions. But instead of creating a new network to have all the tables created and then just overwriting all tables except compressor, we should just check for missing tables in the `convert_format` script and add them there.

Reply:
I see, yes, we should keep this in mind. I will open a new issue.